tests/z80benchtests/CLITests.cs

```diff
@@ -0,0 +1,31 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.IO;
+using System;
+using z80bench;
+
+namespace z80benchtests {
+    [TestClass]
+    public class CLITests {
+        private static readonly string[] _expectedErrorOutput = {
+            "--execute <offset>",
+            "--stack-pointer <offset>",
+            "--max-cycles <count>",
+            "--vram-compare filename[@offset]]"
+        };
+
+        [TestMethod]
+        public void ExitsWithErrorWithoutArguments() {
```

Comment:
Simple tests that asserts that when called with no args:
- The program returns -1
- The first line written to the console should be the usage: line
- All of the options should be present somewhere in the output (no assertion around their description or order)